modules/hal_nordic/nrfs/Kconfig

```diff
@@ -38,8 +38,6 @@ config NRFS
 	bool "nRF Services Support"
 	select NRFS_LOCAL_DOMAIN if (SOC_NRF54H20_CPUAPP || SOC_NRF54H20_CPURAD || SOC_NRF9280_CPUAPP || SOC_NRF9280_CPURAD)
 	depends on HAS_NRFS
-	depends on !MISRA_SANE
```

Comment:
Do you know if this is addressed? https://github.com/nrfconnect/sdk-zephyr/pull/1856#issuecomment-2188854151

Reply:
If that's the case then this needs to be upstreamed.

Reply:
I've tested with the option and I'm able to compile successfully, maybe it's not a problem anymore?